bundles/ApplicationLoggerBundle/src/Maintenance/LogArchiveTask.php

```diff
@@ -63,6 +64,19 @@ public function execute(): void
         $sql = 'SELECT %s FROM '.ApplicationLoggerDb::TABLE_NAME.' WHERE `timestamp` < DATE_SUB(FROM_UNIXTIME('.$timestamp.'), INTERVAL '.$archive_threshold.' DAY)';
 
         if ($db->fetchOne(sprintf($sql, 'COUNT(*)')) > 0) {
+
+            $storageEngine = $this->config['applicationlog']['archive_db_table_storage_engine'];
+            if (!$storageEngine) {
+                // auto-detect if no storage engine is defined in config
+                $engines = $db->fetchFirstColumn('SHOW ENGINES;');
```

Comment:
```suggestion
                $engines = $db->fetchFirstColumn('SELECT Engine FROM information_schema.ENGINES
WHERE Support IN (\'YES\',\ 'DEFAULT\')');
```

The `Support` column could be also set to `DISABLED` or `NO` , which may not be useful for the auto-detection system
<img width="625" height="486" alt="image" src="https://github.com/user-attachments/assets/5f880cd5-7449-4b39-af58-74997f28e4ab" />
source: https://dev.mysql.com/doc/refman/8.4/en/information-schema-engines-table.html